coderd/apikey_test.go

```diff
@@ -144,6 +144,27 @@ func TestTokenUserSetMaxLifetime(t *testing.T) {
 	require.ErrorContains(t, err, "lifetime must be less")
 }
 
+func TestTokenUserSetDefaultLifetime(t *testing.T) {
```

Comment:
This test name is confusing, what about `TestTokenCustomDefaultLifetime` instead

```suggestion
func TestTokenCustomDefaultLifetime(t *testing.T) {
```